content/includes/server-config.md

```diff
@@ -0,0 +1,361 @@
+
+```yaml
+# The config file version. Currently this field is optional but future
+# KES versions will require it. The only valid value is "v1". 
+version: v1
+
+# The TCP address (ip:port) for the KES server to listen on.
+address: 0.0.0.0:7373 # The pseudo address 0.0.0.0 refers to all network interfaces 
+
+admin:
+  # The admin identity identifies the public/private key pair
+  # that can perform any API operation.
+  # The admin account can be disabled by setting a value that
+  # cannot match any public key - e.g. "foobar" or "disabled".
+  identity: c84cc9b91ae2399b043da7eca616048d4b4200edf2ff418d8af3835911db945d
+
+# The TLS configuration for the KES server. A KES server
+# accepts HTTP only over TLS (HTTPS). Therefore, a TLS
+# private key and public certificate must be specified,
+# either here as part of the config file or via CLI arguments.
+tls:
+  key:      ./server.key   # Path to the TLS private key
+  cert:     ./server.cert  # Path to the TLS certificate
+  password: ""             # An optional password to decrypt the TLS private key
+  
+  # An optional path to a file or directory containing X.509 certificate(s).
+  # If set, the certificate(s) get added to the list of CA certificates for
+  # verifying the mTLS certificates sent by the KES clients.
+  #
+  # If empty, the system root CAs will be used.
+  ca:       ""        
+
+  # The TLS proxy configuration. A TLS proxy, like nginx, sits in
+  # between a KES client and the KES server and usually acts as a
+  # load balancer or common endpoint.
+  # All connections from the KES client to the TLS proxy as well
+  # the connections from the TLS proxy to the KES server must be
+  # established over TLS.
+  proxy:
+    # The identities of all TLS proxies directly connected to the
+    # KES server.
+    #
+    # Note that a TLS proxy can act as any identity (including root)
+    # since it can forward arbitrary client certificates. Client certificates
+    # aren't secret information and a malicious TLS proxy can fake any
+    # identity it has seen before. Therefore, its critical that all TLS proxy
+    # identities are secure and trusted servers.
+    identities: []
+    # The HTTP header sent by the TLS proxy to forward certain data
+    # about the client to the KES server.
+    header:
+      # The HTTP header containing the URL-escaped and PEM-encoded
+      # certificate of the kes client forwarded by the TLS proxy.
+      cert: X-Tls-Client-Cert
+
+# The API configuration. The APIs exposed by the KES server can
+# be adjusted here. Each API is identified by its API path.
+#
+# In general, the KES server uses sane defaults for all APIs.
+# Only customize the APIs if there is a real need.
+# 
+# Disabling authentication for an API must be carefully evaluated.
+# One example, when disabling authentication may be justified, would
+# be the liveness and readiness probes in a Kubernetes environment.
```

Comment:
This sentence is a little convoluted, perhaps:
```
# For example, you may choose to disable authentication for a
# Kubernetes readiness probe, but not metrics.

(I'm guessing "Kubernetes readiness probe" is `v1/ready`?)

Reply:
Oh this is directly from the original in the repo. Hmm.

Reply:
Cleaned it up in ours anyway.